boards/arm/stm32f746g_disco/Kconfig.defconfig

```diff
@@ -38,4 +45,17 @@ config PWM_STM32_1
 	default y
 	depends on PWM
 
+config KSCAN
+	default y if LVGL
```

Comment:
I don't think we should enable KSCAN depending on LVGL (unless there is a dependency in the drivers itself, but ideally it should not be the case.).

Reply:
ok, I can agree with that.